### DISCUSSION TITLE
求大佬帮忙！！！umijs4 代理问题

### DISCUSSION THREAD

**Discussion by RiseToTop** [umijs/umi#11798] [Q&A]
我在使用umijs4时，请求后台接口用了代理，但后台服务关闭时整个网站就加载不出来了，一直刷新一直是个空白页，终端报了个Error occurred while proxying request XXXXX  to XXXXX [ETIMEDOUT] (https://nodejs.org/api/errors.html#errors_common_system_errors)

并不是请求接口出错导致的页面崩溃，因为网站就没有加载到请求接口那里就加载不出来了

**Comment by fz6m** [comment]
服务不可用的时候需要自己想办法 mock 数据。

**Reply by RiseToTop** [reply]
我知道，那umijs4就是这样设计的对吗？代理接口不可用网站就加载不了？按道理来说像其他代理库http-proxy-middleware这种，就算服务不可用也不会影响页面啊
